api/tests/devcomp/integration/repositories/passage-repository_test.js

```diff
@@ -35,4 +36,32 @@ describe('Integration | DevComp | Repositories | PassageRepository', function ()
       expect(savedPassage.updatedAt).to.deep.equal(new Date('2023-12-31'));
     });
   });
+
+  describe('#get', function () {
+    describe('when passage exists', function () {
+      it('should return the found passage', async function () {
+        // given
+        const passage = databaseBuilder.factory.buildPassage({ id: 1, moduleId: 'my-module' });
+        await databaseBuilder.commit();
+
+        // when
+        const result = await passageRepository.get({ passageId: 1 });
+
+        // then
+        expect(result).to.be.instanceOf(Passage);
+        expect(result.id).to.equal(passage.id);
+        expect(result.moduleId).to.equal(passage.moduleId);
```

Comment:
Fourni par le test-helpers

```suggestion
        expect(result).to.deepEqualInstance(new Passage({ id, moduleId }));
```